openapi/openapi.json

```diff
@@ -23722,8 +23722,7 @@
       "properties": {
         "dnsManagementPolicy": {
           "description": "dnsManagementPolicy indicates if the lifecycle of the wildcard DNS record associated with the load balancer service will be managed by the ingress operator. It defaults to Managed. Valid values are: Managed and Unmanaged.",
-          "type": "string",
-          "default": ""
```

Comment:
So the problem is that this value here should be Managed?

Reply:
Yes, we need the defaults to be accepted in both OpenAPI and controller-tools, except they use different tags, so right now the default is `""` which is causing validation issues. You must have omitempty to have a non-empty default here as well